js/graviex.js

```diff
@@ -0,0 +1,1012 @@
+            const balance = balances[i];
+            const currencyId = this.safeString (balance, 'currency');
+            const code = this.safeCurrencyCode (currencyId);
+            const account = this.account ();
+            account['free'] = this.safeString (balance, 'balance');
+            account['used'] = this.safeString (balance, 'locked');
+            result[code] = account;
+        }
+        return this.safeBalance (result);
+    }
+
+    async fetchBalance (params = {}) {
+        await this.loadMarkets ();
+        const response = await this.privateGetMembersMe (params);
+        // {
+        //     "sn": "GRAK6GDVXKNVIEX",
+        //     "name": null,
+        //     "email": "XXXXXXXXXXXXX",
+        //     "activated": true,
+        //     "verified": false,
+        //     "accounts_filtered": [
+        //     {
+        //         "currency": "gio",
+        //         "name": "GravioCoin",
+        //         "balance": "0.0",
+        //         "locked": "0.0",
+        //         "status": "ok",
+        //         "is_purged": false,
+        //         "is_coin": true
+        //     }
+        //     ]
+        // }
+        return this.parseBalance (response);
+    }
+
+    async fetchOrderBook (symbol, limit = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchOrderBook() requires a symbol argument');
+        }
+        await this.loadMarkets ();
+        const request = {
+            'market': this.marketId (symbol),
+        };
+        const response = await this.privateGetOrderBook (this.extend (request, params));
+        // {
+        //   "asks": [
+        //     {
+        //       "id": 288323066,
+        //       "at": 1644211934,
+        //       "side": "sell",
+        //       "ord_type": "limit",
+        //       "price": "0.003074559",
+        //       "avg_price": "0.0",
+        //       "state": "wait",
+        //       "market": "ltcbtc",
+        //       "created_at": "2022-02-07T08:32:14+03:00",
+        //       "volume": "10.2778",
+        //       "remaining_volume": "10.2778",
+        //       "executed_volume": "0.0",
+        //       "trades_count": 0,
+        //       "strategy": null
+        //     },
+        //     {
+        //       "id": 288322948,
+        //       "at": 1644211864,
+        //       "side": "sell",
+        //       "ord_type": "limit",
+        //       "price": "0.00307456",
+        //       "avg_price": "0.0",
+        //       "state": "wait",
+        //       "market": "ltcbtc",
+        //       "created_at": "2022-02-07T08:31:04+03:00",
+        //       "volume": "1.0",
+        //       "remaining_volume": "1.0",
+        //       "executed_volume": "0.0",
+        //       "trades_count": 0,
+        //       "strategy": null
+        //     },
+        //   ]
+        // }
+        const timestamp = this.parse8601 (this.safeString (response, 'created_at'));
```

Comment:
Will this timestamp ever exist?  Currently it's not there
![image](https://user-images.githubusercontent.com/43336371/153566178-afa24a48-38be-4cfd-ae2d-75f79ec827c4.png)

Reply:
Okay . Fixed in the latest commit